about.html

```diff
@@ -0,0 +1,183 @@
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+      <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet"
+      />
+      <link rel="stylesheet" href="./assets/css/about.css" />
+      <title>Protoje Munich Concert - About</title>
+    </head>
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <div class="social_bar">
+        <nav class="dsk">
+          <ul>
+            <li>
+              <a href="#"><img src="./assets/images/facebook.png" alt="facebook" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./assets/images/twitter.png" alt="twitter" /></a>
+            </li>
+            <li>
+              <a href="">English</a>
+            </li>
+            <li>
+              <a href="">My Page</a>
+            </li>
+            <li>
+              <a href="">Logout</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+
+      <div class="top_bar">
+        <nav>
+          <a href="index.html"><img src="./assets/images/logo.png" alt="Home" /></a>
+        </nav>
+
+        <nav>
+          <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">Buy Tickets</a></li>
+          </ul>
+        </nav>
+      </div>
+
+      <nav class="hamburger">
+        <a href="#"><img id="hamburguer-icon" src="./assets/images/lista.png" alt="menu" /></a>
+      </nav>
+    </header>
+
+    <div class="hamburguer">
+      <button id="btn-close" type="button" class="btn-close" aria-label="Close"></button>
+      <ul>
+        <li>
+          <a class="hamburguer-link" href="index.html"><h3 class="h3">Home</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="#logo"><h3 class="h3">Concert Logo</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="#pastEvents"><h3 class="h3">Past Concerts</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="about.html"><h3 class="h3">About</h3></a>
+        </li>
+      </ul>
+    </div>
+
+    <main>
+      <section class="heading">
+        <div class="breadcrumbs">
+          <ul class="breadcrumbs">
+            <li>
+              <a href="index.html"
+                ><img class="breadcrumbsImg" src="./assets/images/home.png" alt="home" /><span class="breadcrumbs">&gt;</span></a
+              >
+            </li>
+            <li>
+              <a class="breadcrumbs" href="about.html">About</a>
+            </li>
+          </ul>
+        </div>
+        <h2 class="heading">"Hello Reggae Lover"</h2>
+        <h1>
+          Protoje Reggae <br />
+          Munich Concert <br />
+          2023
+        </h1>
```

Comment:
- [ ] As per the HTML best practices, we should respect the semantic rules, by making sure h1 is above all the other titles so, I suggest to replace the h2 with a p :recycle: